contracts/staking/contracts/src/vaults/EthVault.sol

```diff
@@ -83,21 +91,19 @@ contract EthVault is
         return _balances[owner];
     }
 
-    /// @dev Withdraw an `amount` of ETH to `owner` from the vault.
-    /// @param owner of ETH.
-    /// @param amount of ETH to withdraw.
+    /// @dev Withdraw an `amount` of WETH to `owner` from the vault.
+    /// @param owner of WETH.
+    /// @param amount of WETH to withdraw.
     function _withdrawFrom(address payable owner, uint256 amount)
         internal
     {
-        // update balance
-        // note that this call will revert if trying to withdraw more
-        // than the current balance
+        //Uupdate balance.
```

Comment:
```suggestion
        // Update balance.
```

Reply:
Not touching the vaults anymore since they're getting removed in a coming PR. :man_shrugging: